MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -71,6 +71,11 @@ export host_ip="External_Public_IP"
 export your_no_proxy=${your_no_proxy},"External_Public_IP"
 ```
 
+> Note: The `MAX_IMAGES` environment variable is used to specify the maximum number of images that will be sent from the LVM service to the LLaVA server.
+> If an image list longer than `MAX_IMAGES` is sent to the LVM server, a shortened image list will be sent to the LLaVA service. If the image list
+> needs to be shortened, the most recent images (the ones at the end of the list) are prioritized to send to the LLaVA service. Some LLaVA models have not
+> been trained with multiple images and may lead to inaccurate results. If `MAX_IMAGES` is not set, it will default to `1`.
+
```

Comment:
This is a good note, but I think it should go either under the `export MAX_IMAGES=1` line or at the bottom with the note about `host_ip`.